osbs/http.py

```diff
@@ -34,6 +41,22 @@
 logger = logging.getLogger(__name__)
 
 
+class OSBSRetry(Retry):
+    def is_forced_retry(self, method, status_code, has_retry_after=False):
+        if status_code in HTTP_RETRIES_STATUS_FORCELIST:
+            logger.debug("Server returned returned %s", status_code)
+            return True
+        else:
+            return False
+
+    def increment(self, method=None, url=None, response=None,
```

Comment:
If we must log (must we?) then let's at least accept any `*args, **kwargs` provided, and pass them through to the superclass's implementation.

But I don't think we necessarily need to log.